apps/docs/src/app/core/core-documentation.module.ts

```diff
@@ -131,9 +131,10 @@ import { ModalOpenTemplateExampleComponent } from '../core/component-docs/modal/
 import { ModalContentComponent } from '../core/component-docs/modal/examples/component-as-content/modal-content.component';
 import { ModalComponentAsContentExampleComponent } from '../core/component-docs/modal/examples/component-as-content/modal-component-as-content-example.component';
 import {
-    PanelEdgeBleedExampleComponent,
     PanelExampleComponent
 } from '../core/component-docs/panel/examples/panel-examples.component';
+import { PanelEdgeBleedExampleComponent } from '../core/component-docs/panel/examples/panel-edge-bleed-example.component'
```

Comment:
I guess this file was just auto-formatted - if so then we can do that in a separate PR